packages/xo-web/src/common/intl/messages.js

```diff
@@ -909,8 +909,8 @@ const messages = {
   installAllPatchesOnHostContent:
     'Are you sure you want to install all patches on this host?',
   patchRelease: 'Release',
-  updatePluginNotInstalled:
-    'An error occurred while fetching the patches. Please make sure the updater plugin is installed by running `yum install xcp-ng-updater` on the host.',
```

Comment:
Why did we remove this informative message for a more generic one?

Reply:
Because now, xo-web can get a `'cannot fetch missing patches'` error either because the updater plugin is not installed or because the plugin is busy. Taking [your other comment](https://github.com/vatesfr/xen-orchestra/pull/4519/files#r325669179) into account, should we make everything more generic or more specific? Or a generic error with specific data?

Reply:
IMHO, error types should be more generic, but with enough contextual properties that makes them understandable and debuggable.

On the other hand, errors presented to the users should be clear enough to make them understand what the problem is and how to solve it if they can.